deployment/roles/deploy/templates/worker-is.yml.j2

```diff
@@ -9,7 +9,7 @@ metadata:
   name: worker
 spec:
   tags:
-    - name: {{ deployment }}
+    - name: latest
```

Comment:
@jpopelka I've changed this back, to make the re-deployment work again when the image stream is updated.

I cannot think of a technical reason, why the image stream tag should match the deployment name. Am I missing something?

Reply:
Yes, should be OK.